pkg/meta/interface.go

```diff
@@ -161,6 +166,9 @@ type Attr struct {
 	Parent    Ino  // inode of parent; 0 means tracked by parentKey (for hardlinks)
 	Full      bool // the attributes are completed or not
 	KeepCache bool // whether to keep the cached page or not
+
+	AccessACLId  uint32 // access ACL id (identical ACL rules share the same access ACL ID.)
```

Comment:
`AccessACL` should be enough

Reply:
This is the ACL's ID, not the final value, so I think it should remain unchanged.